content/docs/iac/adopting-pulumi/migrating-to-pulumi/from-cdk.md

```diff
@@ -0,0 +1,469 @@
+        instanceType: props.instanceType,
+      }),
+      ...
+    });
+  }
+}
+
+interface AppStackProps extends cdk.StackProps {
+    cluster: rds.IDatabaseCluster
+}
+
+class AppStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props: AppStackProps) {
+    super(scope, id, props);
+
+    new AppConstruct(this, 'app-construct', {
+        cluster: props.cluster,
+    })
+
+  }
+}
+
+interface MyStageProps extends cdk.StageProps {
+  instanceType: ec2.InstanceType;
+}
+
+class MyStage extends cdk.Stage {
+  constructor(scope: Construct, id: string, props: MyStageProps) {
+    super(scope, id, props);
+      const stateful = new StatefulStack(app, 'stateful-stack', {
+        instanceType: props.instanceType,
+      });
+      new AppStack(app, 'app-stack', {
+          cluster: stateful.cluster,
+      });
+  }
+}
+
+const app = new cdk.App();
+new MyStage(app, 'dev', {
+  instanceType: ec2.InstanceType.of(ec2.InstanceClass.R6G, ec2.InstanceSize.MEDIUM),
+});
+
+new MyStage(app, 'prod', {
+  instanceType: ec2.InstanceType.of(ec2.InstanceClass.R6G, ec2.InstanceSize.XLARGE),
+});
+```
+
+The AWS CDK Stages can be converted to [Pulumi Stacks](https://www.pulumi.com/docs/iac/concepts/stacks/).
+Similar to CDK Stages, Pulumi Stacks can be used to deploy groups of resources to different environments.
+
+In AWS CDK applications differences in configuration between environments are typically configured through input parameters on the Stage.
+In the above example, the `DatabaseCluster` uses a different `InstanceType` between the `dev` and `prod` environments.
+When we convert this application to Pulumi, we will move this configuration from the Stage input properties to [Stack Configuration](https://www.pulumi.com/docs/iac/concepts/config/)
+
+When we convert the AWS CDK application we will:
+
+* Combining the Stacks like we did in the [Multi Stack example](#converting-same-environment-stacks)
+* Remove the use of CDK Stages
+* Read the instance type from the [Pulumi Configuration](https://www.pulumi.com/docs/iac/concepts/config/).
+
+```ts
+import * as pulumi from '@pulumi/pulumi';
+import * as pulumicdk from '@pulumi/cdk';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import * as rds from 'aws-cdk-lib/aws-rds';
+
+const config = new pulumi.Config();
+const instanceType = config.require('instanceType');
+
+interface AppStackProps extends pulumicdk.StackProps {
+  instanceType: ec2.InstanceType;
+}
+
+class AppStack extends pulumicdk.Stack {
+  constructor(scope: pulumicdk.App, id: string, props: AppStackProps) {
+    super(scope, id, props);
+
+    const cluster = new rds.DatabaseCluster(this, 'cluster', {
+      writer: rds.ClusterInstance.provisioned('writer', {
+        instanceType: props.instanceType,
+      }),
+      ...
+    });
+    new AppConstruct(this, 'app-construct', {
+        cluster: cluster,
+    })
+
+  }
+}
+
+const app = new pulumicdk.App('app', (scope: pulumicdk.App) => {
+  new AppStack(app, 'app-stack', {
```

Comment:
This is missing the StatefulStack from the CDK example from above. Is that intentional?